src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -588,7 +591,6 @@
     <Compile Include="System\Xml\Xsl\Runtime\WhitespaceRuleReader.cs" />
     <Compile Include="System\Xml\Xsl\Runtime\XmlAggregates.cs" />
     <Compile Include="System\Xml\Xsl\Runtime\XmlAttributeCache.cs" />
-    <Compile Include="System\Xml\Xsl\Runtime\XmlCollation.cs" />
```

Comment:
>="System\Xml\Xsl\Runtime\XmlCollation.cs" /> [](start = 20, length = 44)

you don't need to remove the whole file. you just need to refactor only the part we need to have in Windows or Linux specific. what you did will cause us to fix stuff twice in this file.